sdk/digitaltwins/Azure.ResourceManager.DigitalTwins/CHANGELOG.md

```diff
@@ -10,6 +23,7 @@
 
 ### Other Changes
```

Comment:
Since 1.1.0-beta.1 has not been released, we should remove this from the changelog. These entries are always auto-updated after publishing a new version.

Reply:
Adding to that: generally, it is better practice to update the changelog, and version in the csproj just before releasing the SDK, in a separate PR. Hence, I would recommend actually reverting the changelog changes, and putting up a separate PR later just with the version bump.

Reply:
If don't want to release 1.1.0-beta.1 but a stable version 1.1.0, please just change the version to `1.1.0` both here and the csproj file.
And also add the description here to explain what kind of changes of this new version.

Reply:
Removed. I will add the changes to the next PR as suggested by @sjiherzig